python/TestHarness/testers/Tester.py

```diff
@@ -72,6 +72,7 @@ def validParams():
         params.addParam('required_objects', [], "A list of required objects that are in the executable.")
         params.addParam('check_input',    False, "Check for correct input file syntax")
         params.addParam('display_required', False, "The test requires and active display for rendering (i.e., ImageDiff tests).")
+        params.addParam('timing',         True, "If True, the test will be allowed to run with the timing flag (i.e. Manually turning on performance logging")
```

Comment:
Need closing parenthesis inside the string

Reply:
(Picky, picky